docs/views.md

```diff
@@ -108,14 +123,11 @@ This is the message: <%= message() %>
 This doesn't correspond to any action in our controller, but we'll exercise it in an `iex` session. At the root of our project, we can run `iex -S mix`, and then explicitly render our template.
 
 ```console
-iex(1)> Phoenix.View.render(HelloPhoenix.Web.PageView, "test.html", %{})
+iex(1)> Phoenix.View.render(HelloPhoenix.Web.PageView, "test.html",
+%{})
   {:safe, [["" | "This is the message: "] | "Hello from the view!"]}
 ```
-As we can see, we're calling `render/3` with the individual view responsible for our test template, the name of our test template, and an empty map representing any data we might have wanted to pass in.
-
-The return value is a tuple beginning with the atom `:safe` and the resultant string of the interpolated template.
-
-"Safe" here means that Phoenix has escaped the contents of our rendered template. Phoenix defines its own `Phoenix.HTML.Safe` protocol with implementations for atoms, bitstrings, lists, integers, floats, and tuples to handle this escaping for us as our templates are rendered into strings.
+As we can see, we're calling `render/3` with the individual view responsible for our test template, the name of our test template, and an empty map representing any data we might have wanted to pass in. The return value is a tuple beginning with the atom `:safe` and the resultant string of the interpolated template. "Safe" here means that Phoenix has escaped the contents of our rendered template. Phoenix defines its own `Phoenix.HTML.Safe` protocol with implementations for atoms, bitstrings, lists, integers, floats, and tuples to handle this escaping for us as our templates are rendered into strings.
```

Comment:
Is it worth mentioning that the tuple returns `:safe` and an io list instead of a string?